File: 02.conditions/12.cpp
#include <iostream>

using namespace std;

int main(){
    string day;
    cin >> day;

    if (day == "monday" || day == "tuesday" ||
        day == "thursday" || day == "friday" || day == "wednesday"){
            cout << "weekday";
    } else if (day == "saturday" || day == "sunday"){
        cout << "weekend";
    }else{
        cout << "invalid day";
    }

    return 0;
}

File: 02.conditions/python/12.py
day = input()

# if day in ["monday", "tuesday", "wednesday", "thursday", "friday"]:
if day == "monday" or day == "tuesday" or day == "wednesday" or day == "thursday" or day == "friday":
    print("weekday")
elif day == "saturday" or day == "sunday":
    print("weekend")
else:
    print("invalid day")
